component/discovery/relabel/relabel.go

```diff
@@ -27,8 +27,8 @@ type Arguments struct {
 	// Targets contains the input 'targets' passed by a service discovery component.
 	Targets []discovery.Target `river:"targets,attr"`
 
-	// The relabelling steps to apply to the each target's label set.
-	RelabelConfigs []*flow_relabel.Config `river:"relabel_config,block,optional"`
+	// The relabelling rules to apply to the each target's label set.
+	RelabelConfigs []*flow_relabel.Config `river:"rule,block,optional"`
```

Comment:
I purposefully left the field names as-is. It should be clearer to the developer that we have to do with relabel_configs here, and when reading the code it makes sense to distinguish between `RelabelConfigs` and `MetricRelabelConfigs`.